apps/codelab/src/app/codelabs/angular/component-tree/component-tree.component.html

```diff
@@ -83,11 +83,7 @@ <h2 i18n>
 
   <!--  Parent/Child COMPONENTS-->
   <div *slide id="parent-child-component" milestone="Component Tree">
-    <h1>
-      <ng-container i18n="@@componentTree">Component Tree</ng-container>
-      /
-      <ng-container i18n>Parent and Child</ng-container>
-    </h1>
+    <codelab-breadcrumb i18n>Parent and Child</codelab-breadcrumb>
```

Comment:
There is no breadcrumb on the previous page: 
http://localhost:4200/angular/component-tree/description

![image](https://user-images.githubusercontent.com/40883170/52531388-b843c600-2d25-11e9-91f8-9718887d62cd.png)